drivers/src/fuse_bank.rs

```diff
@@ -234,9 +234,11 @@ impl FuseBank<'_> {
     ///
     pub fn runtime_svn(&self) -> u32 {
         let soc_ifc_regs = self.soc_ifc.regs();
-        64 - ((soc_ifc_regs.fuse_runtime_svn().at(1).read() as u64) << 32
-            | soc_ifc_regs.fuse_runtime_svn().at(0).read() as u64)
-            .leading_zeros()
+        let fuse: u128 = ((soc_ifc_regs.fuse_runtime_svn().at(3).read() as u128) << 96)
```

Comment:
Can we extract this logic into a function? And add unit tests?

Reply:
fuse_runtime_svn().read() will return a [u32; 4]

Reply:
A function that converts that to u128 would be good.

Reply:
Added.